pymess/backend/push/onesignal.py

```diff
@@ -0,0 +1,56 @@
+from http import HTTPStatus
+from json.decoder import JSONDecodeError
+
+import requests
+from django.utils import timezone
+from django.utils.encoding import force_text
+from onesignal import DeviceNotification, OneSignal
+
+from pymess.backend.push import PushNotificationBackend
+from pymess.config import settings
+from pymess.models import PushNotificationMessage
+from pymess.utils.logged_requests import generate_session
+
+
+class OneSignalPushNotificationBackend(PushNotificationBackend):
+
+    def _is_response_error(self, response):
+        return response.status_code != HTTPStatus.OK
+
+    def _is_response_partial_error(self, response):
+        return not self._is_response_error(response) and response.json().get('errors')
+
+    def _is_errorous(self, response):
```

Comment:
to je strašné slovo. bych použil spíš is_invalid_response

Reply:
:monkah: